DaniloLira/TeamGC
# Hotfix(Remove-History): Removes history and improve code

**What I did:**

- Remove history from main screen

- Removed last news from positive feedback screen

- Refactored code

**How to test:**

- Go to the main screen and see if the history is removed
- Type something true on the text field and see if in the positive feedback appears the last news
